From 551bcbc82b8492d80fd531efd7db00bb69587b6d Mon Sep 17 00:00:00 2001
Message-ID: <551bcbc82b8492d80fd531efd7db00bb69587b6d.1701297273.git.sharlatanus@gmail.com>
In-Reply-To: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
References: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 29 Oct 2023 13:34:07 +0000
Subject: [PATCH v3 32/66] gnu: go-github-com-aws-sdk: Move to (gnu packages
 golang-web).

* gnu/packages/golang.scm (go-github-com-aws-sdk): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: I8e82019c75b115548c2e0eea54b261ff8c3dd4a7
---
 gnu/packages/golang-web.scm | 28 ++++++++++++++++++++++++++++
 gnu/packages/golang.scm     | 28 ----------------------------
 2 files changed, 28 insertions(+), 28 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 0219ffb5e5..352c831e41 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -71,6 +71,34 @@ (define-public go-github-com-andybalholm-cascadia
 the parse trees produced by the html package.")
     (license license:bsd-2)))
 
+(define-public go-github-com-aws-sdk
+  (package
+    (name "go-github-com-aws-sdk")
+    (version "1.35.2")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/aws/aws-sdk-go")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "1ky5lw2s2zpslnnqcs6hgsrwvwbxwgflb5jwf16dd4aga3vrg10c"))))
+    (build-system go-build-system)
+    (arguments
+     '(#:import-path "github.com/aws/aws-sdk-go/aws"
+       #:unpack-path "github.com/aws/aws-sdk-go"))
+    (propagated-inputs
+     (list go-github-com-go-sql-driver-mysql
+           go-github-com-jmespath-go-jmespath
+           go-github-com-pkg-errors
+           go-golang-org-x-net))
+    (home-page "https://github.com/aws/aws-sdk-go")
+    (synopsis "Library to access Amazon Web Services (AWS)")
+    (description
+     "This is the official AWS SDK for the Go programming language.")
+    (license license:asl2.0)))
+
 (define-public go-github-com-aymerick-douceur
   (package
     (name "go-github-com-aymerick-douceur")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index ad2bdf099d..1f1ca8694c 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -3195,34 +3195,6 @@ (define-public go-github-com-hebcal-hebcal-go
 and anniversaries.")
       (license license:gpl2+))))
 
-(define-public go-github-com-aws-sdk
-  (package
-    (name "go-github-com-aws-sdk")
-    (version "1.35.2")
-    (source
-     (origin
-       (method git-fetch)
-       (uri (git-reference
-             (url "https://github.com/aws/aws-sdk-go")
-             (commit (string-append "v" version))))
-       (file-name (git-file-name name version))
-       (sha256
-        (base32
-         "1ky5lw2s2zpslnnqcs6hgsrwvwbxwgflb5jwf16dd4aga3vrg10c"))))
-    (build-system go-build-system)
-    (arguments
-     '(#:import-path "github.com/aws/aws-sdk-go/aws"
-       #:unpack-path "github.com/aws/aws-sdk-go"))
-    (propagated-inputs
-     (list go-github-com-go-sql-driver-mysql
-           go-github-com-jmespath-go-jmespath go-github-com-pkg-errors
-           go-golang-org-x-net))
-    (home-page "https://github.com/aws/aws-sdk-go")
-    (synopsis "Library to access Amazon Web Services (AWS)")
-    (description
-     "This is the official AWS SDK for the Go programming language.")
-    (license license:asl2.0)))
-
 (define-public go-github-com-go-jose-go-jose-v3
   (package
     (inherit go-gopkg-in-square-go-jose-v2)
-- 
2.41.0

